readme.md

```diff
@@ -48,6 +48,7 @@ free to, just please open an issue, or get in contact with me, so I can add it t
 | DragonflyBSD | [![DPorts](https://repology.org/badge/version-for-repo/dports/spotify-qt.svg?header=DPorts)](https://github.com/DragonFlyBSD/DPorts/blob/master/audio/spotify-qt/Makefile) | [ehaupt](https://github.com/ehaupt) |
 | Void Linux | [![Void Linux](https://repology.org/badge/version-for-repo/void_x86_64/spotify-qt.svg?header=Void%20Linux)](https://github.com/void-linux/void-packages/blob/master/srcpkgs/spotify-qt/template) | [abenson](https://github.com/abenson) |
 | nixpkgs | [![nixpkgs unstable](https://repology.org/badge/version-for-repo/nix_unstable/spotify-qt.svg?header=nixpkgs%20unstable)](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/spotify-qt/default.nix) | [karthikiyengar](https://github.com/karthikiyengar)
+| openSUSE Tumbleweed | [![openSUSE Multimedia:Apps Tumbleweed package](https://repology.org/badge/version-for-repo/opensuse_multimedia_apps_tumbleweed/spotify-qt.svg)](https://software.opensuse.org//download.html?project=multimedia%3Aapps&package=spotify-qt) | [KaratekHD](https://github.com/KaratekHD)
```

Comment:
That is a very long header xd
Maybe set the header to "openSUSE" or "openSUSE Tumbleweed"?

Reply:
Done :+1: 
I also removed the word "Tumbleweed", as the link also works on Leap.